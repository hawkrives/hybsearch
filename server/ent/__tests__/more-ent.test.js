/* eslint-env jest */
require('jest-specific-snapshot')

const { removeFastaIdentifiers } = require('../../formats')
const { parseFasta } = require('../../formats/fasta/parse')
const { pruneOutliers } = require('../../lib/prune-newick')
const { removeCircularLinks } = require('../../pipeline/lib')

const ent = require('../index')
const { parse: newickToJson } = require('../../newick')
const fs = require('fs')
const path = require('path')

const base = path.join(__dirname, '..', '..', '__supporting__', 'input')
const files = fs
	.readdirSync(base)
	.filter(f => f.endsWith('.tree'))
	.map(f => f.replace(/(.*)\.tree$/, '$1'))

for (const file of files) {
	let content = fs.readFileSync(path.join(base, file) + '.tree', 'utf-8')
	let alignedFasta = fs.readFileSync(path.join(base, file) + '.fasta', 'utf-8')
	let serializedTree = JSON.stringify(newickToJson(content))

	test(file, () => {
		let { prunedNewick } = pruneOutliers(
			JSON.parse(serializedTree),
			alignedFasta
		)

		let serializedNewick = JSON.stringify(prunedNewick)

		let nonmonoInfo = removeCircularLinks(
			ent.strictSearch(JSON.parse(serializedNewick), alignedFasta)
		)

		let monophyleticFasta = removeFastaIdentifiers(alignedFasta, nonmonoInfo)
		let results = parseFasta(monophyleticFasta).map(s => s.species)

		expect(results).toMatchSpecificSnapshot(`./__snapshots__/${file}.hybsnap`)
	})
}
