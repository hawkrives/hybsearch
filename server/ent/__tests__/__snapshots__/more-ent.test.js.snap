// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`coyote-test 1`] = `
Array [
  "Canis_latrans__EU400576",
  "Canis_latrans__EU400575",
  "Canis_latrans__EU400574",
  "Canis_latrans__EU400573",
  "Canis_latrans__EU400572",
  "Canis_latrans__EU400571",
  "Canis_latrans__EU400570",
  "Canis_latrans__EU400569",
  "Canis_latrans__EU400568",
  "Canis_latrans__EU400567",
  "Canis_latrans__EU400566",
  "Canis_latrans__EU400565",
  "Canis_latrans__EU400564",
  "Canis_latrans__EU400563",
  "Canis_latrans__EU400562",
  "Canis_latrans__EU400561",
  "Canis_latrans__EU400560",
  "Canis_latrans__EU400559",
  "Canis_latrans__EU400558",
  "Canis_latrans__EU400557",
  "Canis_latrans__EU400556",
  "Canis_latrans__EU400555",
  "Canis_latrans__EU400554",
  "Canis_latrans__EU400553",
  "Canis_latrans__EU400552",
  "Canis_latrans__EU400551",
  "Canis_latrans__EU400550",
  "Canis_latrans__EU400549",
  "Canis_latrans__EU400548",
  "Canis_latrans__EU400547",
  "Canis_latrans__EU400546",
  "Canis_latrans__EU400545",
  "Canis_latrans__EU400544",
  "Canis_lupus__AF115703",
  "Canis_lupus__AF115702",
  "Canis_lupus__AF115701",
  "Canis_lupus__AF115700",
  "Canis_lupus__AF115699",
  "Canis_lupus__AF115698",
  "Canis_lupus__AF115697",
  "Canis_lupus__AF115696",
  "Canis_lupus__AF115695",
  "Canis_lupus__AF115694",
  "Canis_lupus__AF115693",
  "Canis_lupus__AF115692",
  "Canis_lupus__AF115691",
  "Canis_lupus__AF115690",
  "Canis_lupus__AF115689",
  "Canis_lupus__AF115688",
  "Canis_lupus__AF115687",
]
`;

exports[`emydura 1`] = `
Array [
  "Emydura subglobosa__KC755190",
  "Emydura victoriae__KC755189",
  "Emydura victoriae__KC755188",
  "Emydura tanybaraga__KC755187",
  "Emydura tanybaraga__KC755186",
  "Emydura subglobosa__KC755185",
  "Emydura macquarii__KC755183",
]
`;

exports[`emydura-short 1`] = `
Array [
  "Emydura subglobosa__KC755190",
  "Emydura victoriae__KC755189",
  "Emydura victoriae__KC755188",
  "Emydura tanybaraga__KC755187",
  "Emydura tanybaraga__KC755186",
  "Emydura subglobosa__KC755185",
  "Emydura macquarii__KC755183",
]
`;

exports[`kino 1`] = `
Array [
  "Sternotherus carinatus__JN860673",
  "Kinosternon subrubrum__KF301391",
  "Kinosternon subrubrum__KF301390",
  "Kinosternon sonoriense__KF301389",
  "Kinosternon scorpioides__KF301387",
  "Kinosternon scorpioides__KF301386",
  "Kinosternon scorpioides__KF301385",
  "Kinosternon oaxacae__KF301384",
  "Kinosternon leucostomum__KF301383",
  "Kinosternon integrum__KF301382",
  "Kinosternon integrum__KF301381",
  "Kinosternon integrum__KF301380",
  "Kinosternon hirtipes__KF301379",
  "Kinosternon herrerai__KF301378",
  "Kinosternon herrerai__KF301377",
  "Kinosternon flavescens__KF301376",
  "Kinosternon durangoense__KF301375",
  "Kinosternon dunni__KF301374",
  "Kinosternon creaseri__KF301373",
  "Kinosternon chimalhuaca__KF301372",
  "Kinosternon baurii__KF301371",
  "Kinosternon arizonense__KF301370",
  "Kinosternon angustipons__KF301369",
  "Kinosternon alamosae__KF301368",
  "Kinosternon acutum__KF301367",
  "Kinosternon acutum__KF301366",
  "Sternotherus odoratus__KF301365",
  "Sternotherus minor__KF301364",
  "Sternotherus minor__KF301363",
  "Sternotherus depressus__KF301362",
  "Sternotherus carinatus__KF301361",
  "Staurotypus triporcatus__KF301360",
  "Staurotypus salvinii__KF301359",
  "Claudius angustatus__KF301358",
]
`;

exports[`plaamyd 1`] = `
Array [
  "Amyda ornata__LM537490",
  "Amyda ornata__LM537482",
  "Amyda ornata__LM537478",
  "Amyda ornata__LM537462",
  "Amyda cartilaginea__LM537466",
  "Palea steindachneri__AY259552",
  "Palea steindachneri__AY743417",
]
`;

exports[`smallerlepus 1`] = `
Array [
  "Lepus mandshuricus__HM233091",
  "Lepus mandshuricus__HM233090",
  "Lepus mandshuricus__HM233089",
  "Lepus mandshuricus__HM233088",
  "Lepus mandshuricus__HM233077",
  "Lepus mandshuricus__HM233014",
  "Lepus mandshuricus__HM233011",
  "Lepus mandshuricus__HM233010",
  "Lepus mandshuricus__HM232996",
  "Lepus mandshuricus__HM232995",
  "Lepus mandshuricus__HM232994",
  "Lepus mandshuricus__HM232993",
  "Lepus mandshuricus__HM232992",
  "Lepus mandshuricus__HM232991",
  "Lepus brachyurus__AB058616",
  "Lepus brachyurus__AB058615",
  "Lepus timidus__AB058607",
  "Lepus mandshuricus__DQ793161",
  "Lepus coreanus__AB687534",
  "Lepus coreanus__AB687533",
  "Lepus coreanus__AB687532",
  "Lepus coreanus__AB687531",
  "Lepus timidus__AB687530",
  "Lepus timidus__AB687529",
  "Lepus timidus__AB687528",
  "Lepus timidus__AB687527",
  "Lepus timidus__AB687526",
  "Lepus timidus__AB687525",
  "Lepus timidus__AB687524",
]
`;

exports[`trio171sequence 1`] = `
Array [
  "Pelodiscus_sinensis__AB904729",
  "Pelodiscus_sinensis__AB904728",
  "Pelodiscus_sinensis__AB904727",
  "Pelodiscus_sinensis__AB904726",
  "Pelodiscus_maackii__AB904725",
  "Pelodiscus_maackii__AB904724",
  "Pelodiscus_maackii__AB904723",
  "Pelodiscus_maackii__AB904722",
  "Pelodiscus_maackii__AB904721",
  "Pelodiscus_maackii__AB904720",
  "Amyda_ornata__LM537490",
  "Amyda_ornata__LM537482",
  "Amyda_ornata__LM537478",
  "Amyda_ornata__LM537462",
  "Amyda_cartilaginea__LM537466",
  "Pelodiscus_sinensis__FR851459",
  "Nilssonia_nigricans__HE801760",
  "Nilssonia_nigricans__HE801759",
  "Nilssonia_nigricans__HE801758",
  "Nilssonia_nigricans__HE801757",
  "Nilssonia_hurum__HE801756",
  "Nilssonia_hurum__HE801755",
  "Nilssonia_hurum__HE801754",
  "Nilssonia_hurum__HE801753",
  "Nilssonia_hurum__HE801752",
  "Nilssonia_gangetica__HE801751",
  "Nilssonia_gangetica__HE801750",
  "Nilssonia_gangetica__HE801749",
  "Nilssonia_gangetica__HE801748",
  "Nilssonia_gangetica__HE801747",
  "Nilssonia_gangetica__HE801746",
  "Nilssonia_gangetica__HE801745",
  "Nilssonia_gangetica__HE801744",
  "Nilssonia_gangetica__HE801743",
  "Nilssonia_gangetica__HE801742",
  "Nilssonia_formosa__HE801741",
  "Nilssonia_formosa__HE801740",
  "Pelochelys_cantorii__JN232533",
  "Lissemys_punctata__JN232531",
  "Chitra_indica__JN232529",
  "Nilssonia_hurum__JN232528",
  "Nilssonia_nigricans__JN232527",
  "Cycloderma_aubryi__FR850656",
  "Cycloderma_aubryi__FR850655",
  "Cyclanorbis_senegalensis__FR850654",
  "Lissemys_scutata__FR850653",
  "Lissemys_scutata__FR850652",
  "Lissemys_scutata__FR850651",
  "Lissemys_scutata__FR850650",
  "Lissemys_punctata__FR850649",
  "Lissemys_punctata__FR850648",
  "Lissemys_punctata__FR850647",
  "Lissemys_punctata__FR850646",
  "Lissemys_punctata__FR850645",
  "Lissemys_punctata__FR850644",
  "Lissemys_punctata__FR850643",
  "Lissemys_punctata__FR850642",
  "Lissemys_punctata__FR850641",
  "Lissemys_punctata__FR850640",
  "Lissemys_punctata__FR850639",
  "Lissemys_punctata__FR850638",
  "Lissemys_punctata__FR850637",
  "Lissemys_punctata__FR850636",
  "Lissemys_punctata__FR850635",
  "Lissemys_punctata__FR850634",
  "Lissemys_punctata__FR850633",
  "Lissemys_punctata__FR850632",
  "Lissemys_punctata__FR850631",
  "Lissemys_punctata__FR850630",
  "Lissemys_punctata__FR850629",
  "Lissemys_punctata__FR850628",
  "Lissemys_punctata__FR850627",
  "Lissemys_punctata__FR850626",
  "Lissemys_punctata__FR850625",
  "Lissemys_punctata__FR850624",
  "Lissemys_punctata__FR850623",
  "Lissemys_punctata__FR850622",
  "Lissemys_punctata__FR850621",
  "Lissemys_punctata__FR850620",
  "Lissemys_punctata__FR850619",
  "Lissemys_punctata__FR850618",
  "Lissemys_punctata__FR850617",
  "Lissemys_punctata__FR850616",
  "Lissemys_punctata__FR850615",
  "Lissemys_punctata__FR850614",
  "Lissemys_punctata__FR850613",
  "Lissemys_punctata__FR850612",
  "Lissemys_punctata__FR850611",
  "Lissemys_punctata__FR850610",
  "Lissemys_punctata__FR850609",
  "Lissemys_punctata__FR850608",
  "Lissemys_punctata__FR850607",
  "Pelodiscus_maackii__FM999011",
  "Cyclanorbis_elegans__AY259570",
  "Cyclanorbis_senegalensis__AY259569",
  "Lissemys_punctata__AY259568",
  "Lissemys_scutata__AY259567",
  "Cycloderma_aubryi__AY259566",
  "Cycloderma_frenatum__AY259565",
  "Trionyx_triunguis__AY259564",
  "Chitra_vandijki__AY259563",
  "Chitra_chitra__AY259562",
  "Chitra_indica__AY259561",
  "Pelochelys_cantorii__AY259560",
  "Pelochelys_bibroni__AY259559",
  "Apalone_spinifera__AY259558",
  "Apalone_spinifera__AY259557",
  "Apalone_mutica__AY259556",
  "Apalone_ferox__AY259555",
  "Rafetus_euphraticus__AY259554",
  "Pelodiscus_sinensis__AY259553",
  "Palea_steindachneri__AY259552",
  "Dogania_subplana__AY259551",
  "Nilssonia_gangetica__AY259549",
  "Nilssonia_hurum__AY259548",
  "Nilssonia_formosa__AY259547",
  "Palea_steindachneri__AY743417",
  "Pelodiscus_axenaria__AY583693",
  "Pelodiscus_sinensis__AY583692",
  "Nilssonia_gangetica__AM495217",
  "Nilssonia_gangetica__AM495216",
  "Nilssonia_gangetica__AM495215",
  "Nilssonia_nigricans__AM495236",
  "Nilssonia_nigricans__AM495235",
  "Nilssonia_nigricans__AM495234",
  "Nilssonia_nigricans__AM495233",
  "Nilssonia_nigricans__AM495232",
  "Nilssonia_nigricans__AM495231",
  "Nilssonia_nigricans__AM495230",
  "Nilssonia_nigricans__AM495229",
  "Nilssonia_nigricans__AM495228",
  "Nilssonia_nigricans__AM495227",
  "Nilssonia_leithii__AM495226",
  "Nilssonia_leithii__AM495225",
  "Nilssonia_hurum__AM495224",
  "Nilssonia_hurum__AM495223",
  "Nilssonia_hurum__AM495222",
  "Nilssonia_hurum__AM495221",
  "Nilssonia_hurum__AM495220",
  "Nilssonia_hurum__AM495219",
  "Nilssonia_hurum__AM495218",
  "Nilssonia_gangetica__AM495214",
  "Nilssonia_gangetica__AM495213",
  "Nilssonia_gangetica__AM495212",
  "Nilssonia_gangetica__AM495211",
  "Nilssonia_gangetica__AM495210",
  "Nilssonia_gangetica__AM495209",
  "Nilssonia_gangetica__AM495208",
]
`;

exports[`trionychcytb 1`] = `
Array [
  "Acomys_dimidiatus__Z96060",
  "Acomys_dimidiatus__Z96061",
  "Acomys_dimidiatus__Z96062",
  "Acomys_spinosissimus__Z96067",
  "Acomys_spinosissimus__Z96068",
  "Trionyx_triunguis__AY259564",
  "Amyda_ornata__LM537462",
  "Amyda_ornata__LM537478",
  "Amyda_ornata__LM537482",
  "Amyda_ornata__LM537490",
  "Balaenoptera_bonaerensis__X75581",
  "Apalone_spinifera__AY259558",
  "Amphistichus_argenteus__JN125234",
  "Amphistichus_argenteus__JN125235",
  "Amphistichus_argenteus__JN125236",
  "Pelochelys_bibroni__AY259559",
  "Amyda_cartilaginea__AY259550",
  "Amyda_cartilaginea__JN232534",
  "Amyda_cartilaginea__LM537465",
  "Amyda_cartilaginea__LM537466",
  "Amyda_cartilaginea__LM537467",
  "Amyda_cartilaginea__LM537476",
  "Amyda_cartilaginea__LM537480",
  "Amyda_cartilaginea__LM537481",
  "Amyda_cartilaginea__LM537483",
  "Amyda_cartilaginea__LM537484",
  "Amyda_cartilaginea__LM537485",
  "Amyda_cartilaginea__LM537491",
  "Amyda_cartilaginea__LM537492",
  "Amyda_cartilaginea__LM537493",
  "Amyda_cartilaginea__LM537494",
  "Balaena_mysticetus__X75588",
  "Lissemys_scutata__AY259567",
  "Lissemys_scutata__FR850650",
  "Lissemys_scutata__FR850651",
  "Lissemys_scutata__FR850652",
  "Lissemys_scutata__FR850653",
  "Amphistichus_koelzi__JN125237",
  "Amphistichus_koelzi__JN125238",
  "Amphistichus_koelzi__JN125239",
  "Pelochelys_cantorii__AY259560",
  "Pelochelys_cantorii__JN232533",
  "Capra_ibex__AJ010055",
  "Pelodiscus_sinensis__AY259553",
  "Pelodiscus_sinensis__AY583692",
  "Pelodiscus_sinensis__JN860674",
  "Pelodiscus_sinensis__FR851459",
  "Pelodiscus_sinensis__AB904726",
  "Pelodiscus_sinensis__AB904727",
  "Pelodiscus_sinensis__AB904728",
  "Pelodiscus_sinensis__AB904729",
  "Chioninia_delalandii__AJ305008",
  "Chioninia_delalandii__AJ305009",
  "Chioninia_delalandii__AJ305010",
  "Chioninia_delalandii__AJ305011",
  "Chioninia_delalandii__AJ305012",
  "Chioninia_delalandii__AJ305013",
  "Chioninia_delalandii__AJ305014",
  "Chioninia_delalandii__AJ305015",
  "Chioninia_delalandii__AJ305016",
  "Chioninia_delalandii__AJ305017",
  "Chioninia_delalandii__AJ305018",
  "Chitra_chitra__AY259562",
  "Chitra_indica__AY259561",
  "Chitra_indica__JN232529",
  "Chitra_vandijki__AY259563",
  "Paracentrotus_lividus__AH001273",
  "Cyclanorbis_elegans__AY259570",
  "Cyclanorbis_senegalensis__AY259569",
  "Cyclanorbis_senegalensis__FR850654",
  "Cycloderma_aubryi__AY259566",
  "Cycloderma_aubryi__FR850655",
  "Cycloderma_aubryi__FR850656",
  "Cycloderma_frenatum__AY259565",
  "Dogania_subplana__AY259551",
  "Acomys_cahirinus__Z96051",
  "Acomys_cahirinus__Z96052",
  "Acomys_cahirinus__Z96053",
  "Rafetus_euphraticus__AY259554",
  "Acomys_ignitus__Z96063",
  "Acomys_ignitus__Z96064",
  "Apalone_ferox__AY259555",
  "Acomys_russatus__Z96065",
  "Acomys_russatus__Z96066",
  "Eschrichtius_robustus__X75585",
  "Apalone_spinifera__AY259557",
  "Megaptera_novaeangliae__X75584",
  "Pelodiscus_axenaria__AY583693",
  "Lissemys_punctata__AY259568",
  "Mustela_nivalis__AM932882",
  "Mustela_nivalis__AM932883",
  "Mustela_nivalis__AM932884",
  "Mustela_nivalis__AM932885",
  "Lissemys_punctata__FR850607",
  "Lissemys_punctata__FR850608",
  "Lissemys_punctata__FR850609",
  "Lissemys_punctata__FR850610",
  "Lissemys_punctata__FR850611",
  "Lissemys_punctata__FR850612",
  "Lissemys_punctata__FR850613",
  "Lissemys_punctata__FR850615",
  "Lissemys_punctata__FR850616",
  "Lissemys_punctata__FR850617",
  "Lissemys_punctata__FR850618",
  "Lissemys_punctata__FR850619",
  "Lissemys_punctata__JN232531",
  "Lissemys_punctata__FR850614",
  "Lissemys_punctata__FR850620",
  "Lissemys_punctata__FR850621",
  "Lissemys_punctata__FR850622",
  "Lissemys_punctata__FR850623",
  "Lissemys_punctata__FR850624",
  "Lissemys_punctata__FR850625",
  "Lissemys_punctata__FR850626",
  "Lissemys_punctata__FR850627",
  "Lissemys_punctata__FR850628",
  "Lissemys_punctata__FR850629",
  "Lissemys_punctata__FR850630",
  "Lissemys_punctata__FR850631",
  "Lissemys_punctata__FR850632",
  "Lissemys_punctata__FR850633",
  "Lissemys_punctata__FR850634",
  "Lissemys_punctata__FR850635",
  "Lissemys_punctata__FR850636",
  "Lissemys_punctata__FR850637",
  "Lissemys_punctata__FR850638",
  "Lissemys_punctata__FR850639",
  "Lissemys_punctata__FR850640",
  "Lissemys_punctata__FR850641",
  "Lissemys_punctata__FR850642",
  "Lissemys_punctata__FR850643",
  "Lissemys_punctata__FR850644",
  "Lissemys_punctata__FR850645",
  "Lissemys_punctata__FR850646",
  "Lissemys_punctata__FR850647",
  "Lissemys_punctata__FR850648",
  "Lissemys_punctata__FR850649",
  "Balaenoptera_acutorostrata__X75753",
  "Nilssonia_formosa__AY259547",
  "Nilssonia_formosa__HE801740",
  "Nilssonia_formosa__HE801741",
  "Nilssonia_gangetica__AY259549",
  "Nilssonia_gangetica__AM495208",
  "Nilssonia_gangetica__AM495209",
  "Nilssonia_gangetica__AM495210",
  "Nilssonia_gangetica__AM495211",
  "Nilssonia_gangetica__AM495212",
  "Nilssonia_gangetica__AM495213",
  "Nilssonia_gangetica__AM495214",
  "Nilssonia_gangetica__AM495215",
  "Nilssonia_gangetica__AM495216",
  "Nilssonia_gangetica__AM495217",
  "Nilssonia_gangetica__HE801742",
  "Nilssonia_gangetica__HE801743",
  "Nilssonia_gangetica__HE801744",
  "Nilssonia_gangetica__HE801745",
  "Nilssonia_gangetica__HE801746",
  "Nilssonia_gangetica__HE801747",
  "Nilssonia_gangetica__HE801748",
  "Nilssonia_gangetica__HE801749",
  "Nilssonia_gangetica__HE801750",
  "Nilssonia_gangetica__HE801751",
  "Nilssonia_hurum__AY259548",
  "Nilssonia_hurum__AM495218",
  "Nilssonia_hurum__AM495219",
  "Nilssonia_hurum__AM495220",
  "Nilssonia_hurum__AM495221",
  "Nilssonia_hurum__AM495222",
  "Nilssonia_hurum__AM495223",
  "Nilssonia_hurum__AM495224",
  "Nilssonia_hurum__JN232528",
  "Nilssonia_hurum__HE801752",
  "Nilssonia_hurum__HE801753",
  "Nilssonia_hurum__HE801754",
  "Nilssonia_hurum__HE801755",
  "Nilssonia_hurum__HE801756",
  "Nilssonia_leithii__AM495225",
  "Nilssonia_leithii__AM495226",
  "Nilssonia_nigricans__AM495227",
  "Nilssonia_nigricans__AM495228",
  "Nilssonia_nigricans__AM495229",
  "Nilssonia_nigricans__AM495230",
  "Nilssonia_nigricans__AM495231",
  "Nilssonia_nigricans__AM495232",
  "Nilssonia_nigricans__AM495233",
  "Nilssonia_nigricans__AM495234",
  "Nilssonia_nigricans__AM495235",
  "Nilssonia_nigricans__AM495236",
  "Nilssonia_nigricans__AM495237",
  "Nilssonia_nigricans__JN232527",
  "Nilssonia_nigricans__HE801757",
  "Nilssonia_nigricans__HE801758",
  "Nilssonia_nigricans__HE801759",
  "Nilssonia_nigricans__HE801760",
  "Eubalaena_glacialis__X75587",
  "Capra_nubiana__AJ009879",
  "Pelodiscus_maackii__FM999011",
  "Pelodiscus_maackii__AB904720",
  "Pelodiscus_maackii__AB904721",
  "Pelodiscus_maackii__AB904722",
  "Pelodiscus_maackii__AB904723",
  "Pelodiscus_maackii__AB904724",
  "Pelodiscus_maackii__AB904725",
  "Pongo_pygmaeus__X97717",
  "Balaenoptera_edeni__X75583",
  "Caperea_marginata__X75586",
  "Pnoepyga_pusilla__JX518496",
  "Amphistichus_rhodoterus__JN125240",
  "Amphistichus_rhodoterus__JN125241",
  "Amphistichus_rhodoterus__JN125242",
  "Macaca_mulatta__AF378365",
  "Corvus_frugilegus__Y16885",
  "Balaenoptera_borealis__X75582",
  "Hyperprosopon_ellipticum__JN125246",
  "Apalone_mutica__AY259556",
  "Anser_caerulescens__X77190",
  "Mus_minutoides__FN985221",
  "Mus_minutoides__FN985222",
  "Mus_minutoides__FN985223",
  "Mus_minutoides__FN985224",
  "Capra_pyrenaica__AJ010047",
  "Capra_pyrenaica__AJ010048",
  "Capra_pyrenaica__AJ010049",
  "Capra_pyrenaica__AJ010050",
  "Capra_pyrenaica__AJ010051",
  "Capra_pyrenaica__AJ010052",
  "Capra_pyrenaica__AJ010053",
  "Capra_pyrenaica__AJ010054",
  "Capra_pyrenaica__AJ010056",
  "Physeter_catodon__X75589",
  "Hyperprosopon_anale__JN125243",
  "Hyperprosopon_anale__JN125244",
  "Hyperprosopon_anale__JN125245",
  "Mus_musculoides__Z96069",
  "Hyperprosopon_argenteum__JN125250",
  "Palea_steindachneri__AY259552",
  "Palea_steindachneri__AY743417",
]
`;

exports[`vulpes 1`] = `
Array [
  "Vulpes_cana__KU378587",
  "Vulpes_cana__KU378586",
  "Vulpes_cana__KU378585",
  "Vulpes_cana__KU378584",
  "Vulpes_cana__KU378583",
  "Vulpes_cana__KU378582",
  "Vulpes_cana__KU378581",
  "Vulpes_vulpes__KU378580",
  "Vulpes_vulpes__KU378579",
  "Vulpes_vulpes__KU378578",
  "Vulpes_vulpes__KU378577",
  "Vulpes_vulpes__KU378576",
  "Vulpes_vulpes__KU378575",
  "Vulpes_vulpes__KU378574",
  "Vulpes_vulpes__KU378573",
  "Vulpes_vulpes__KU378572",
  "Vulpes_vulpes__KU378571",
  "Vulpes_vulpes__KU378570",
  "Vulpes_vulpes__KU378569",
  "Vulpes_vulpes__KU378568",
  "Vulpes_vulpes__KU378567",
  "Vulpes_vulpes__KU378566",
  "Vulpes_vulpes__KU378565",
  "Vulpes_vulpes__KU378564",
  "Vulpes_vulpes__KU378563",
  "Vulpes_vulpes__KU378562",
  "Vulpes_vulpes__KU378561",
  "Vulpes_vulpes__KU378560",
  "Vulpes_vulpes__KU378559",
  "Vulpes_vulpes__KU378557",
  "Vulpes_vulpes__KU378556",
  "Vulpes_vulpes__KU378555",
  "Vulpes_vulpes__KU378552",
  "Vulpes_vulpes__KU378549",
  "Vulpes_vulpes__KU378548",
  "Vulpes_vulpes__KU378547",
  "Vulpes_vulpes__KU378546",
  "Vulpes_vulpes__KU378545",
  "Vulpes_vulpes__KU378542",
  "Vulpes_vulpes__KU378541",
  "Vulpes_vulpes__KU378540",
  "Vulpes_vulpes__KU378539",
  "Vulpes_vulpes__KU378538",
  "Vulpes_vulpes__KU378537",
  "Vulpes_vulpes__KU378536",
  "Vulpes_vulpes__KU378535",
  "Vulpes_vulpes__KU378533",
  "Vulpes_vulpes__KU378532",
  "Vulpes_vulpes__KU378530",
  "Vulpes_vulpes__KU378529",
  "Vulpes_vulpes__KU378528",
  "Vulpes_vulpes__KU378527",
  "Vulpes_vulpes__KU378526",
  "Vulpes_vulpes__KU378525",
  "Vulpes_vulpes__KU378524",
  "Vulpes_vulpes__KU378523",
  "Vulpes_vulpes__KU378522",
  "Vulpes_vulpes__KU378521",
  "Vulpes_vulpes__KU378520",
  "Vulpes_vulpes__KU378518",
  "Vulpes_vulpes__KU378517",
  "Vulpes_vulpes__KU378516",
  "Vulpes_vulpes__KU378515",
  "Vulpes_vulpes__KU378514",
  "Vulpes_vulpes__KU378513",
  "Vulpes_vulpes__KU378512",
  "Vulpes_vulpes__KU378511",
  "Vulpes_vulpes__KU378510",
  "Vulpes_vulpes__KU378509",
  "Vulpes_vulpes__KU378508",
  "Vulpes_vulpes__KU378507",
  "Vulpes_vulpes__KU378506",
  "Vulpes_vulpes__KU378505",
  "Vulpes_vulpes__KU378504",
  "Vulpes_vulpes__KU378503",
  "Vulpes_vulpes__KU378502",
  "Vulpes_vulpes__KU378501",
  "Vulpes_vulpes__KU378500",
  "Vulpes_vulpes__KU378498",
  "Vulpes_vulpes__KU378497",
  "Vulpes_vulpes__KU378495",
  "Vulpes_vulpes__KU378494",
  "Vulpes_vulpes__KU378491",
  "Vulpes_corsac__KU378490",
  "Vulpes_corsac__KU378489",
  "Vulpes_corsac__KU378488",
  "Vulpes_corsac__KU378487",
  "Vulpes_rueppellii__KU378373",
  "Vulpes_rueppellii__KU378372",
  "Vulpes_rueppellii__KU378371",
  "Vulpes_rueppellii__KU378370",
  "Vulpes_rueppellii__KU378369",
  "Vulpes_rueppellii__KU378368",
  "Vulpes_vulpes__EF689062",
  "Vulpes_vulpes__EF689061",
  "Vulpes_vulpes__EF689060",
  "Vulpes_vulpes__EF689059",
  "Vulpes_vulpes__EF689058",
  "Vulpes_vulpes__AY928669",
  "Vulpes_vulpes__DQ498127",
  "Vulpes_vulpes__DQ498126",
  "Vulpes_vulpes__DQ498125",
  "Vulpes_vulpes__DQ498124",
  "Vulpes_lagopus__LT560065",
  "Vulpes_lagopus__LT560053",
  "Vulpes_lagopus__LT560041",
  "Vulpes_lagopus__LT560029",
  "Vulpes_lagopus__LT560017",
  "Vulpes_lagopus__LT560005",
  "Vulpes_lagopus__LT559993",
  "Vulpes_lagopus__LT559981",
  "Vulpes_lagopus__LT559969",
  "Vulpes_lagopus__LT559957",
  "Vulpes_lagopus__LT559945",
  "Vulpes_lagopus__LT559933",
  "Vulpes_lagopus__LT559921",
  "Vulpes_lagopus__LT559909",
  "Vulpes_lagopus__LT559897",
  "Vulpes_lagopus__LT559885",
  "Vulpes_lagopus__LT559873",
  "Vulpes_lagopus__LT559861",
  "Vulpes_lagopus__LT559849",
  "Vulpes_lagopus__LT559837",
  "Vulpes_lagopus__LT559825",
  "Vulpes_lagopus__LT559813",
  "Vulpes_lagopus__LT559801",
  "Vulpes_lagopus__LT559789",
  "Vulpes_lagopus__LT559777",
  "Vulpes_lagopus__LT559765",
  "Vulpes_lagopus__LT559753",
  "Vulpes_lagopus__LT559741",
  "Vulpes_lagopus__LT559729",
  "Vulpes_lagopus__LT559717",
  "Vulpes_lagopus__LT559705",
  "Vulpes_lagopus__LT559693",
  "Vulpes_lagopus__LT559681",
  "Vulpes_lagopus__LT559669",
  "Vulpes_lagopus__LT559657",
  "Vulpes_lagopus__LT559645",
  "Vulpes_lagopus__LT559633",
  "Vulpes_lagopus__LT559621",
  "Vulpes_lagopus__LT559609",
  "Vulpes_lagopus__LT559597",
  "Vulpes_lagopus__LT559585",
  "Vulpes_lagopus__LT559573",
  "Vulpes_lagopus__LT559561",
  "Vulpes_lagopus__LT559549",
  "Vulpes_lagopus__LT559537",
  "Vulpes_lagopus__LT559525",
  "Vulpes_lagopus__LT559513",
  "Vulpes_lagopus__LT559501",
  "Vulpes_lagopus__LT559489",
  "Vulpes_macrotis__AH005827",
  "Vulpes_vulpes__KM657043",
  "Vulpes_vulpes__KM657042",
  "Vulpes_vulpes__KM657041",
  "Vulpes_vulpes__KM657040",
  "Vulpes_vulpes__KM657039",
  "Vulpes_vulpes__KM657038",
  "Vulpes_vulpes__JQ003578",
  "Vulpes_vulpes__JQ003577",
  "Vulpes_vulpes__JN652618",
  "Vulpes_vulpes__JN652617",
  "Vulpes_vulpes__JN652616",
  "Vulpes_vulpes__JN652615",
  "Vulpes_vulpes__JN652614",
  "Vulpes_vulpes__JN652613",
  "Vulpes_vulpes__JN652612",
  "Vulpes_vulpes__JN652611",
  "Vulpes_vulpes__JN652610",
  "Vulpes_vulpes__JN652609",
  "Vulpes_vulpes__JN652608",
  "Vulpes_vulpes__JN652607",
  "Vulpes_vulpes__JN652606",
  "Vulpes_vulpes__JN652605",
  "Vulpes_vulpes__JN652604",
  "Vulpes_vulpes__JN652603",
  "Vulpes_vulpes__JX013648",
  "Vulpes_vulpes__JX013647",
  "Vulpes_vulpes__JX013646",
  "Vulpes_macrotis__JF489127",
]
`;
